APM PHP Onboarding (#7103)
DataDog/documentation: content/en/tracing/setup/php.md

export DD_TRACE_PHP_BIN=$(which php-fpm7)
```

Restart PHP (PHP-FPM or the Apache SAPI) and then visit a tracing-enabled endpoint of your application. View the [APM UI][8] to see the traces.

**Note**: It might take a few minutes before traces appear in the UI. If traces still do not appear after a few minutes, [run the dd-doctor.php diagnostic script][9] from the host machine to help identify any issues.

If you can't find your distribution, you can [manually install][10] the PHP extension.

## Automatic Instrumentation

Tracing is automatically enabled by default. Once the extension is installed, **ddtrace** traces your application and sends traces to the Agent.

> We support all web framework out of the box. For specific web frameworks, listed below, we provide even more internal details automatically for you.

Reviewer: s/all web framework/all web frameworks/

#### Web Framework Compatibility

> By default we **support all web frameworks** out of the box. This means that you will be able to see traces which

Reviewer 2: ```suggestion
By default Datadog **supports all web frameworks** out of the box, which allows you to see traces for all supported libraries irrespective of the web framework, 

**Example with PDO library used with Magento 2.0 framework** 
![image](https://user-images.githubusercontent.com/5246917/81242503-2c6d2800-8fc2-11ea-8afd-10914f08bb63.png)
```

| Slim           | 2.x           | _Coming Soon_   |
| Yii            | 1.1           | _Coming Soon_   |

> _Restrictions for specific supported web frameworks_

Reviewer 2: Let's not call out "restrictions"

## Upgrading

To upgrade the PHP tracer, [download the latest release][7] and follow the same steps as [installing the extension](#install-the-extension).

_Note_: if you are using second level caching in opcache (i.e. you have set the parameter `opcache.file_cache`) then you want to remove the cache folder.

## Removing

In order to remove the PHP tracer:

- if you are serving your app using php-fpm, stop the php-fpm service, otherwise stop Apache web server.
- unlink files `98-ddtrace.ini` and `99-ddtrace-custom.ini` from your php configuration folder.

> - if you are using php-fpm, you only need to restart php-fpm service, otherwise restart Apache web server.

Reviewer 3: ```suggestion
3. For php-fpm, restart php-fpm service, otherwise restart the Apache web server.
```

> - Note: if you are using second level caching in opcache (i.e. you have set the parameter `opcache.file_cache`) then you want to remove the cache folder.

Reviewer 3: ```suggestion

**Note**: If you are using second level caching in OPcache by setting the parameter `opcache.file_cache`, remove the cache folder.
```
